src/main/java/com/exadel/discount/mapper/CountryMapper.java

```diff
@@ -0,0 +1,17 @@
+package com.exadel.discount.mapper;
+
+import com.exadel.discount.dto.CountryDTO;
```

Comment:
Please, move your DTO classes into folders dto/country and dto/city accordinally:

import com.exadel.discount.dto.country.CountryDTO;
I need this )

Reply:
If we have just 1 dto for city, there's no need to place them in separate folders